### ISSUE TITLE
QUID des fichiers liés lors de l'import Biodiv'Sport>Geotrek Admin

### ISSUE THREAD

**Issue by thomaspnrhj** [GeotrekCE/Geotrek-admin#2654]
A noter pour plus tard pour les évolutions liées à Biodiv'sport : 
Lors de l'import Biodiv'Sport>Geotrek Admin, si un fichier lié est associé à la zone (photo par exemple), il n'est pas importé dans le Geotrek.

**Comment by camillemonchicourt** [comment]
Oui, comme c'est le cas pour les autres objets (randos, POI, contenus touristiques...), il faudrait ajouter les fichiers liés dans la route des zones sensibles de l'API V2, et enrichir le parser d'import avec les fichiers attachés.
